proto/internalapi/central/sensor_events.proto

```diff
@@ -80,6 +81,8 @@ message SensorEvent {
         storage.ImageIntegration        image_integration       = 13;
         ReprocessDeploymentRisk         reprocess_deployment    = 17;
         AlertResults                    alert_results           = 18;
+        // XXX: Is this really needed?
+        storage.ProcessListeningOnPort  process_listening_on_port = 26;
```

Comment:
AFAICT you're not using `SensorEvent` messages in the pipeline. Unless you're planning on sending and handling `SensorEvents` in the future I think this line can be removed.